### PR TITLE
docs(template): add documentation on manifest file

### PR DESCRIPTION
**Pull request by demetriusfeijoo**
## What?
Add details about the manifest file setup inside our templates.

## Why?

JIRA: EXT-1881

### CODE REVIEW

File: packages/cli/templates/js/README.md [hunk 1]
```diff
@@ -24,6 +24,48 @@ Deploy the field plugin with the CLI. Issue a [personal access token](https://ap
...
+
+```json
+{
+  "options": []
+}
+```
+
+That's all there is to it.
+
+The options list within this file should consist of key-value objects representing the essential options required for your field plugin to function correctly, along with their corresponding values. This is an example of how it should be structured:
```
[the remark this answers is not in the document]
Comment (demetriusfeijoo) [reply packages/cli/templates/js/README.md:48]: Yeah, I was not sure about keeping this part as well, @eunjae-lee.

Thanks for the suggestion, I'm going to commit it. 🙌

File: packages/cli/templates/vue2/README.md [hunk 2]
```diff
@@ -24,6 +24,35 @@ Deploy the field plugin with the CLI. Issue a [personal access token](https://ap
...
+  ]
+}
+```
+
+Now, you just need to access these options in your code like in the example below:
+
+```js
+const { type, data, actions } = useFieldPlugin()
+
+console.log(data.options.myPluginInitialValue)
```
Comment (eunjae-lee) [inline packages/cli/templates/vue2/README.md:53]:
```suggestion
<script>
export default {
  ...
  mixins: [fieldPluginMixin],
  ...
}
</script>

<template>
  <div v-if="plugin.type === 'loaded'">
    <pre>{{ plugin.data.options.myPluginInitialValue }}</pre>
  </div>
</template>
```

File: packages/cli/templates/js/README.md [hunk 3]
```diff
@@ -24,6 +24,35 @@ Deploy the field plugin with the CLI. Issue a [personal access token](https://ap
...
+  ]
+}
+```
+
+Now, you just need to access these options in your code like in the example below:
+
+```js
+const { type, data, actions } = useFieldPlugin()
+
+console.log(data.options.myPluginInitialValue)
```
Comment (eunjae-lee) [inline packages/cli/templates/js/README.md:53]:
```suggestion
console.log(data.options.myPluginInitialValue)
```

**Comment by demetriusfeijoo** [comment]
Nice catch, @eunjae-lee. 
These modifications were really a must. Thanks 🚀

**Review by eunjae-lee — APPROVED** [review]
Looks great!

**Pull request merged**